File: 03_Basic_Recursion/01_Print_Name_NTimes.cs
// Print name n times.

Console.Write("Enter value of N: ");
int N = Convert.ToInt32(Console.ReadLine());
f(1,N);


void f(int i, int N)
{
    if(i > N) return;
    
    Console.WriteLine("Aashish");
    f(i+1, N);
}

File: 03_Basic_Recursion/01_Print_Name_NTimes.cpp
#include <bits/stdc++.h>
using namespace std;

void f(int i, int N)
{
    if (i > N)
        return;

    cout << "Aashish" << endl;
    f(i + 1, N);
}

int main()
{
    int N;
    cout << "Enter value of N: ";
    cin >> N;

    f(1, N);

    return 0;
}